kythe/go/indexer/indexer.go

```diff
@@ -151,18 +155,37 @@ func (pi *packageImporter) Import(importPath string) (*types.Package, error) {
 
 	// Fetch the required input holding the package for this import path, and
 	// load its export data for use by the type resolver.
-	if fi := pi.fileMap[importPath]; fi != nil {
-		data, err := pi.fetcher.Fetch(fi.Path, fi.Digest)
-		if err != nil {
-			return nil, fmt.Errorf("fetching %q (%s): %v", fi.Path, fi.Digest, err)
-		}
-		r, err := gcexportdata.NewReader(bytes.NewReader(data))
-		if err != nil {
-			return nil, fmt.Errorf("reading export data in %q (%s): %v", fi.Path, fi.Digest, err)
+	fi := pi.fileMap[importPath]
+
+	// Check all possible vendor/ paths for missing package file
+	if fi == nil {
```

Comment:
It doesn't need to be since a missing map key results in a zero value (`nil` in this case) and we know that we never put a `nil` into the map (if we did, the existing code below would panic).